### DISCUSSION TITLE
OSM to OSMAnd process

### DISCUSSION THREAD

**Discussion by srroof** [osmandapp/OsmAnd#11385] [Q&A] [answered]
I've been using OSMAnd for years and have contributed to OSM, now I'd like learn about the process connecting the two. What is the process that creates OSMAnd maps from the OSM source? Mostly I'm curious, but if possible, I might consider helping create OSMAnd maps for my  region.

**Comment by stephan75** [comment]
So you already tried Osmands mapcreator tool? What are your results so far?

**Comment by vshcherb** [comment, marked as answer]
Have you checked Technical documentation - https://docs.osmand.net/en/main@latest/development/map-creation/create-offline-maps-yourself. We keep adding new information there.
